src/qililab/qprogram/qblox_compiler.py

```diff
@@ -184,11 +188,19 @@ def traverse(block: Block):
             for bus in self._buses.keys() & times_of_flight.keys():
                 self._buses[bus].time_of_flight = times_of_flight[bus]
 
+        # Pre-processing: Set markers ON/OFF
+        for bus in self._buses:
+            mask = markers[bus] if markers is not None and bus in markers else "0000"
+            self._buses[bus].qpy_sequence._program.blocks[0].append_component(QPyInstructions.SetMrk(int(mask, 2)))
+            self._buses[bus].qpy_sequence._program.blocks[0].append_component(QPyInstructions.UpdParam(4))
```

Comment:
Are those small waits taken into account while syncing? Or do they not count for anything because are in the beginning and end?

Reply:
They don't make any difference tbh, since they are in the beginning and the end and are added to all buses.

Reply:
I added them to `static_duration` property for completeness.